doc/classes/CanvasItem.xml

```diff
@@ -658,7 +658,7 @@
 		</signal>
 		<signal name="visibility_changed">
 			<description>
-				Emitted when the [CanvasItem]'s visibility changes, either because its own [member visible] property changed or because the visility of one of its [CanvasItem] ancestors changed (equivalent to a change in [method is_visible_in_tree]).
+				Emitted when [member visible] changes or when [method is_visible_in_tree] would return a different value.
```

Comment:
Seems correct and concise now, but I somehow dislike the wording regarding `is_visible_in_tree`. What about something like this (it's based on the wording used in `hidden`'s description):
```suggestion
				Emitted when [member visible] changes or when the return value of [method is_visible_in_tree] changes.
```
?
(opinions welcomed; that's only a suggestion)

Reply:
Same thoughts as prior comment.
```suggestion
				Emitted when the [CanvasItem]'s visibility changes, either because its own [member visible] property changed or because its visibility in the tree changed (see [method is_visible_in_tree]).
```

Reply:
'because its visibility in the tree changed' is a good suggestion for this particular description. I do agree that 'when the return value changes' somehow feels less of a state change than 'its visibility changes' and is therefore more fitting.

Reply:
@Mickeon I still think the _"when the [CanvasItem]'s visibility changes"_ part is unclear / easy to misinterpret, the same way as I've explained in https://github.com/godotengine/godot/pull/97223#discussion_r1771560521.

I like the _"its visibility in the tree changed (see [method is_visible_in_tree])"_ wording though (aka normal sentence + see method link).

Reply:
Then how about...

`Emitted when the [CanvasItem]'s [member visible] property changes or when its visibility in the tree changed (see [method is_visible_in_tree]).`

Reply:
I like it! (not sure about @Mickeon :stuck_out_tongue:)

Reply:
I've read, but in a nutshell "When its visibility changes" is a good way to put it. The rest of the sentence elaborates on what it means, which mitigates this ambiguity.

Reply:
Alright, then I'm sticking mostly to what it was earlier, but the latter half of the sentence improved.